components/infobox/wikis/starcraft2/infobox_league_custom.lua

```diff
@@ -67,7 +67,31 @@ function CustomLeague:createWidgetInjector()
 end
 
 function CustomInjector:parse(id, widgets)
-	if id == 'gamesettings' then
+	if id == 'type' then
+		local value = tostring(_args.type):lower()
+		local category
+		if value == 'offline' then
+			category = 'Offline Tournaments'
+		elseif value == 'online' then
+			category = 'Online Tournaments'
+		elseif value:match('online') and value:match('offline') then
+			category = 'Online/Offline Tournaments'
+		elseif value == 'weekly online' then
+			category = 'Weekly Online Tournaments'
+		else
+			category = 'Unknown Type Tournaments'
+		end
+		if not String.isEmpty(_args.type) then
+			return {
+				Cell{
+					name = 'Type',
+					content = {
+						_args.type:sub(1,1):upper() .. _args.type:sub(2) .. '[[Category:' .. category .. ']]'
```

Comment:
mw.language:ucfirst()

Reply:
also something to consider for the commons version ;)
fixed for this PR^^

Reply:
#845